data/Language/JavaScript.yaml

```diff
@@ -19,3 +19,11 @@ delimiters:
   - round_braces
   - square_braces
   - backticks
+datatypes:
+  - boolean_js
+  - null_js
+  - undefined_js
+  - number_js
```

Comment:
please create data/DataType/number_js.yaml to describe this data type .

`data/DataType/integer_toml.yaml` is a good example to follow

You can find the technical details at https://developer.mozilla.org/en-US/docs/Glossary/number

Reply:
what max should i choose?
Number.MAX_VALUE or  Number.MIN_SAFE_INTEGER

and should I create string_js.yaml? or just number_js.yaml only

Reply:
Someone else can do string_js